### DISCUSSION TITLE
Game crashes when in museum

### DISCUSSION THREAD

**Discussion by ExperiencersInternational** [RedShyGuy/Vapecord-ACNL-Plugin#113] [Q&A] [answered]
I'm not filing this as a issue, since I have a suspicion my SD card is dodgy, I did swap out adapters but it seems to still be an issue, can someone clarify that this error is an SD card issue?

![IMG_20220903_170301](https://user-images.githubusercontent.com/56035537/188278823-024fe57c-a1eb-4430-9c2d-078bfd20c709.jpg)

3DS completely freezes so I can't take a crash dump, and I don't have Discord so I can't report it there. With this adapter, the only other crashes I've had so far are in CTGP-7, and that looks to be SD related, with a previous adapter, I had issues with the home menu not loading properly, various ndsbootstrap crashes etc.

Crash info:

Vapecord ACNL Plugin | Version: 1.9.6
Exception Type: Data Abort
Process: GARDEN
Title ID: 0004000000086400

R00:   0000004C
R01:   0FFFFC98
R02:   0000004C
R03:   00000000
R04:   0000004C
R05:   00000000
R06:   00000000
R07:   00000000
R08:   30470598
R09:   00000001
R10:   00000000
R11:   0FFFFDC8
R12:   00000000

SP:    0FFFFC70
LR:    004EDBAC
PC:    00135C74

CPSR:  20000010
FPEXC: 00000000
FAR:   0000004C

**Comment by RedShyGuy** [comment]
Please give more details what you did before the crash happened as well as a list of all activated cheats

**Reply by ExperiencersInternational** [reply]
I was walking to the museum to donate a giant trevally that I had just gotten from Tortimer Island.

I had a few activated, the only one I used was the particle changer changing it to 012A. I believe I also had the message extender, flower waterer, instant item dropper etc enabled

**Reply by RedShyGuy** [reply]
Alright okay, I will look into it

**Reply by ExperiencersInternational** [reply]
I did run H2testw yesterday, and it came out as fine. I'm wondering whether calling the SD '3DS' had any issue, I don't see any mention of it causing an issue on 3ds.hacks.guide but I know for consoles like the Wii U, calling the SD 'WiiU' can cause issues. I've renamed my SD to 'threeDS', hoping I won't keep encountering these errors.

**Reply by ExperiencersInternational** [reply]
I still seem to be running into this issue, like I've mentioned I've ran H2testw and there's been no issue with the SD. However, still occurs in other titles such as CTGP-7.

![1663689802532218891272296546638](https://user-images.githubusercontent.com/56035537/191308337-d6f08600-6ba5-40ec-b735-aa0673c36dba.jpg)

**Comment by ExperiencersInternational** [comment]
Just going to close this thread given I no longer use the plugin and it's been over 2 years since the last reply but I have also heard of other users experiencing this issue too which likely means it isn't an SD card issue.

(btw I don't know how best to say this, but as a heads up, Pretendo is releasing a plugin for RCE patches in ACNL and that will render users with Vapecord unavailable to go online unless Vapecord implements the patches, you might want to look into it before it goes public).

**Comment by FoofooTheGuy** [comment, marked as answer]
it was fixed in d187e0a9f0d66e4d65f1b934cdb6e469cb8aea49 btw.
I think everybody forgot github has this menu
